var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -88,16 +104,32 @@ def configure_args(self):
         else:
             config_args.append('--disable-mt')
 
+        if '+cma' in spec:
+            config_args.append('--enable-cma')
+        else:
+            config_args.append('--disable-cma')
+
         if '+paramter_checking' in spec:
             config_args.append('--enable-params-check')
         else:
             config_args.append('--disable-params-check')
 
+        # Activate SIMD based on properties of the target
+        if self.spec.target >= 'zen':
+            config_args.append('--with-avx')
```

Comment:
Does this not apply to Intel architectures or piledriver (bdver2) and above?

Reply:
Agree, please have a look at latest code changes.